### ISSUE TITLE
Indeces，labels data tree not show.

### ISSUE THREAD

**Issue by Mashull** [albertoventurini/graphdb-intellij-plugin#21]
plugin version: 0.2.1
Neo4j version: 3.5.26
IDEA version: 2022.2.3 Ultimate
---
Connected test successfully...

![image](https://user-images.githubusercontent.com/118787336/203912447-74203e1e-60bd-40b5-b581-9ee098950f67.png)

**Comment by albertoventurini** [comment]
Thank you @Mashull for reporting this issue. I suspect it is due to the version of Neo4j. Is upgrading to 4.4+ or 5.x an option?

In any case, if there is still a use base for Neo4j 3.x, this needs to be fixed. I will fix it in a new release.

**Comment by Mashull** [comment]
Hi，@albertoventurini 
Our products are based on neo4j 3.x, upgrading may bring some unnecessary problems. If possible, I except the plugin can adapt to this version.^-^

**Comment by albertoventurini** [comment]
Hi @Mashull 

I created a new bugfix release which should address this issue. Please see https://github.com/albertoventurini/graphdb-intellij-plugin/releases/tag/v0.2.2 if you want to download and install the plugin as a zip file.

The new release should be available from JetBrains marketplace later today or tomorrow.

**Comment by Mashull** [comment]
It worked ! Thank you @albertoventurini 

But indexes is also miss, like that:
![image](https://user-images.githubusercontent.com/118787336/203932886-7c3c23a0-b80e-42bb-8b56-0c59c7ee53a6.png)

![image](https://user-images.githubusercontent.com/118787336/203932947-03361bdf-399b-47a0-89d7-a97921d9aebd.png)

**Comment by Mashull** [comment]
And plugin console is empty.
![image](https://user-images.githubusercontent.com/118787336/203933410-214e0c5b-1a73-40fe-9889-3efd547539a0.png)
![image](https://user-images.githubusercontent.com/118787336/203933434-9548187f-d2a5-4b97-9fe0-123b15044ad8.png)

**Comment by albertoventurini** [comment]
Thank you @Mashull. Yes, for older versions of Neo4j, indices are retrieved in a different way. Supporting that will require more work. I've created a separate issue (#22) to track that work.

The release I just created (v0.2.2) should at least allow you to retrieve labels and relationships, and use them for Cypher autocompletion. I thought this is an important feature, and I wanted to get it fixed it for Neo4j 3.x as soon as possible.

I will work on indices, constraints, and procedures at a different time.
